.azure-pipelines/recover_testbed/recover_testbed.py

```diff
@@ -40,9 +40,9 @@ def recover_via_console(sonichost, conn_graph_facts, localhost, mgmt_ip, image_u
 
         if type in ["arista"]:
             posix_shell_aboot(dut_console, mgmt_ip, image_url)
-        # elif type in ["Cisco"]:
-        #     return
-        elif type in ["mellanox", "nexus", "acs"]:
+        elif type in ["nexus"]:
```

Comment:
Can you rename variable `type` to something like `device_type`? The reason is that it conflicts with python's builtin `type`.

Reply:
done